### PR TITLE
review styling in progress

### PR DESCRIPTION
**Pull request by Jaguirre123**

### CODE REVIEW

File: client/Components/App.test.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { shallow } from "enzyme";
-import App from "./App.jsx";
+import App from "./App";
```
Comment (Megaloden106) [inline client/Components/App.test.js]: Would it have been better to leave the .jsx? I'm not completely sure if it matters too much.

File: client/Components/App.test.js [hunk 2]
```diff
@@ -14,3 +14,11 @@ describe("<App />", () => {
 //     const wrapper = shallow(<App />).dive();
 //   });
 // });
+
+// describe("App", () => {
+//   it("componentDidMount should fetch, and put resturants in state if ok", async () => {
+//     const renderedComponent = await shallow(<App />);
+//     await renderedComponent.update();
```
Comment (Megaloden106) [inline client/Components/App.test.js]: can also try done() if you are still having async issues. syntax can be found in the documentation if needed, but both works

File: client/Components/App.test.js (hunk 1 above)
Comment (Jaguirre123) [reply client/Components/App.test.js]: I don't even remember changing thats but will fix asap.

File: client/Components/App.test.js (hunk 2 above)
Comment (Jaguirre123) [reply client/Components/App.test.js]: Thank you

**Pull request merged**